environments/common/inventory/group_vars/all/prometheus.yml

```diff
@@ -50,4 +50,11 @@ prometheus_scrape_configs_default:
   scrape_interval: 30s
   scrape_timeout: 20s
 
+- job_name: "slurm_exporter"
+  scrape_interval: 30s
+  scrape_timeout: 30s
+  static_configs:
+    - targets:
+      - "{{ openhpc_slurm_control_host }}:{{ slurm_exporter_port }}"
```

Comment:
If `--listen-address` is configurable, then `{{ openhpc_slurm_control_host }}` would need to be updated too.

Reply:
listen address is not configurable now we're using the unit file from the rpm